clients/client-sdk/src/clients/identity.ts

```diff
@@ -293,4 +354,110 @@ export class IdentityClient extends BaseClient {
   async verifyJwt(jwt: VerifyJwtBody): Promise<{ isValid: boolean; error?: string }> {
     return await this.post(`${this.baseUrl}/authentication/verify-jwt`, jwt);
   }
+
+  private async createIdentity(): Promise<IdentityKeys> {
```

Comment:
could you name this `encodeIdentityKeys` and pass the generated identity into the method? would make more sense here. So remove `const identity = await this.generateIdentity();`, call it before and pass it as param into this method